### ISSUE TITLE
[v6] Link field not keyboard-accessible

### ISSUE THREAD

**Issue by medienbaecker** [getkirby/kirby#8274]
## Description

The link field's "remove" action for pages or files is not keyboard-accessible. This was already the case in v5 but now we're also missing the focus outline:

https://github.com/user-attachments/assets/37aa36b5-8201-4528-8067-b1d61830aefd

**Expected behavior**  
Seeing the focus outline, being able to focus the × or triggering the "remove" action by pressing the whole thing.

## To reproduce

1. Add a link field
2. Select a page/file
3. Try to remove it via the keyboard

## Your setup
**Kirby Version**  
6.0.0-alpha.3

**Your system (please complete the following information)**
 - OS: macOS
 - Browser: Chromium
 - Version: 146.0.7680.177

**Comment by medienbaecker** [comment]
Thanks for the fix, the outline is back!

But how do I actually remove the selection with the keyboard? I guess this also applies to the tags field.

**Comment by distantnative** [comment]
It might need the other fix coming in a PR as well but then delete/backspace while focused.
